src/harness/fake_sas.py

```diff
@@ -229,6 +229,9 @@ def BlacklistByFccIdAndSerialNumber(self, request):
   def PreloadRegistrationData(self, request):
     pass
 
+  def InjectExZone(self, request, ssl_cert=None, ssl_key=None):
```

Comment:
Please spell out "Exclusion" for readability.

Reply:
Added the change